### PR TITLE
[EventHub] az eventhubs eventhub create/update: Change documentation of destination_name

### PR DESCRIPTION
**Pull request by brunomartinspro**
**Description of PR (Mandatory)**  
We need to know the allowed variables for running the command with --destination-name.
This only changes the documentation on [https://docs.microsoft.com/en-us/cli/azure/eventhubs/eventhub](https://docs.microsoft.com/en-us/cli/azure/eventhubs/eventhub)

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ x] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

#10031

### CODE REVIEW

**Comment by yonzhan** [comment]
add to S167

**Comment by brunomartinspro** [comment]
> @brunomartinspro Hi, this PR title is not in the right format, please modify the PR title and description. For details, refer to: [doc link](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests)

Hi! Is it okay now? If not what should I change it to?

**Comment by brunomartinspro** [comment]
@zhoxing-ms @yonzhan All builds passed..

**Comment by brunomartinspro** [comment]
@zhoxing-ms @yonzhan the builds have passed! 👍

**Comment by zhoxing-ms** [comment]
@brunomartinspro If your requirements are related to specific commands, you can add the corresponding command to the title, like: 
> [IoT] iot hub: Support IotHub in the profile of 2019-03-01-hybrid (#12069)

**Comment by brunomartinspro** [comment]
> @brunomartinspro If your requirements are related to specific commands, you can add the corresponding command to the title, like: 
> 
> > [IoT] iot hub: Support IotHub in the profile of 2019-03-01-hybrid (#12069) 

Okay but since I only updated the documentation, is it not enough “[EventHub] Change  documentation of destination_name”?
I just added more text to the description.

**Comment by zhoxing-ms** [comment]
You might write it like this: `[EventHub] az eventhubs eventhub create/update: Change documentation of destination_name`.  It's up to you~

**Comment by brunomartinspro** [comment]
> @brunomartinspro You might write it like this: `[EventHub] az eventhubs eventhub create/update: Change documentation of destination_name`. It's up to you~

@zhoxing-ms i updated to your recommendation, thanks!

**Comment by brunomartinspro** [comment]
@v-Ajnava any news on this?

File: src/azure-cli/azure/cli/command_modules/eventhubs/_params.py
Comment (v-Ajnava) [inline src/azure-cli/azure/cli/command_modules/eventhubs/_params.py:82]: currently we are not supporting 'EventHubArchive.AzureDataLake' in clients, its a swagger issue.  we will need to remove Datalake part

**Comment by yonzhan** [comment]
add to S170

**Comment by yonzhan** [comment]
add to S171

**Comment by yonzhan** [comment]
add to S172

**Comment by zhoxing-ms** [comment]
@brunomartinspro Hi, could you please take a look at the new request from v-Ajnava? Is there any update?

**Comment by zhoxing-ms** [comment]
Because there is no reply for a long time, so I tag it `Do Not Merge` first.

**Comment by brunomartinspro** [comment]
@zhoxing-ms @yonzhan Added the pull request recommendations.

**Comment by zhoxing-ms** [comment]
@v-Ajnava Could you please help to review it again

**Review by MyronFanQiu — APPROVED** [review]

**Review by Juliehzl — APPROVED** [review]

**Comment by Juliehzl** [comment]
@v-Ajnava please help take a look at the PR. Without your approval, we could not merge the PR.

**Pull request merged**
